x11-misc/viewglob/files/viewglob-2.0.4-C23.patch

```diff
@@ -0,0 +1,56 @@
+https://bugs.gentoo.org/944019
+https://bugs.gentoo.org/921573
+Fix implicit defines in configure by defaulting to GNU_SOURCES, set include
+ordering straight for implicit define in code and fix sizes/declarations
+for C23
+--- a/configure.ac
++++ b/configure.ac
+@@ -1,6 +1,7 @@
+ dnl Process this file with autoconf to produce a configure script.
+ AC_INIT([viewglob], [2.0.4])
+ AM_INIT_AUTOMAKE
++AC_USE_SYSTEM_EXTENSIONS
```

Comment:
Good! Note that when using this, you need to make sure that config.h is consistently included in all files _before_ any other includes. Can you check that here?